SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UNR-3761/SpatialTestReplicatedStartupActor/SpatialTestReplicatedStartupActor.cpp

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+
+#include "SpatialTestReplicatedStartupActor.h"
+#include "SpatialFunctionalTestFlowController.h"
+#include "Kismet/GameplayStatics.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/TestActors/ReplicatedTestActorBase.h"
+#include "SpatialFunctionalTestFlowController.h"
+#include "Net/UnrealNetwork.h"
+#include "ReplicatedStartupActorPlayerController.h"
+
+
+/**
+ * This test automates the ReplicatedStartupActor gym. The gym was used for:
+ * - QA workflows Test Replicated startup actors are correctly spawned on all clients
+ * - To support QA test case "C1944 Replicated startup actors are correctly spawned on all clients"
+ * NOTE: This test requires a specific Map with a ReplicatedTestActorBase placed on the map and in the interest of the players and
+ *		 a custom GameMode and PlayerController, trying to run this test on a different Map will make it fail.
+ *
+ * The flow is as follows:
+ * - Setup:
+ *  - Each client sets its reference to the replicated actor and sends a server RPC.
+ * - Test:
+ *  - Each client tests that the server has a valid reference to its replicated actor.
+ */
+
+ASpatialTestReplicatedStartupActor::ASpatialTestReplicatedStartupActor()
+	: Super()
+{
+	Author = "Andrei";
+	Description = TEXT("Test Replicated Startup Actor");
+
+	bIsValidReference = false;
+}
+
+void ASpatialTestReplicatedStartupActor::GetLifetimeReplicatedProps(TArray<FLifetimeProperty>& OutLifetimeProps) const
+{
+	Super::GetLifetimeReplicatedProps(OutLifetimeProps);
+
+	DOREPLIFETIME(ASpatialTestReplicatedStartupActor, bIsValidReference);
+}
+
+void ASpatialTestReplicatedStartupActor::BeginPlay()
+{
+	Super::BeginPlay();
+
+	AddStep(TEXT("SpatialTestReplicatedStartupActorClientsSetup"),FWorkerDefinition::AllClients, [this](ASpatialFunctionalTest* NetTest)
```

Comment:
```suggestion
	AddStep(TEXT("SpatialTestReplicatedStartupActorClientsSetup"), FWorkerDefinition::AllClients, [this](ASpatialFunctionalTest* NetTest)
```
Ultra nit

Reply:
I am amazed you spotted it 🤣! Thanks for pointing it out, will change it now!